muDIC/post/viz.py

```diff
@@ -428,9 +428,27 @@ def show(self, field="displacement", component=(0, 0), frame=0, quiverdisp=False
                 plt.quiver(self.fields.coords()[0, 0, :, :, frame], self.fields.coords()[0, 1, :, :, frame],
                            self.fields.disp()[0, 0, :, :, frame], self.fields.disp()[0, 1, :, :, frame],**kwargs)
             else:
-                plt.contourf(xs, ys, fvar, 50, **kwargs)
-                plt.colorbar()
-        plt.show()
+                im = plt.contourf(xs, ys, fvar, **kwargs)
+                m = plt.cm.ScalarMappable(cmap=im.get_cmap())
+                m.set_array(fvar)
+                m.set_clim(*im.get_clim())
+                cbar = plt.colorbar(m)
+        if 'vmax' in kwargs or 'vmin' in kwargs :
+           print("The colorbar has been cropped")
+        else :
+            print("The colorbar has been established automatically")
+
+        plt.title(title,loc='center')
```

Comment:
I get an error here because of `title`:
```
Traceback (most recent call last):
  File "quick_start.py", line 53, in <module>
    viz.show(field="displacement", component = (1,1), frame=-1)
  File "/home/buster/Repositories/ets-lipec/muDIC/muDIC/post/viz.py", line 441, in show
    plt.title(title,loc='center')
NameError: name 'title' is not defined
```
It seems like `title` was never defined. Could you double check here @NicolasTHU

Reply:
I made an other PR which normally works. It's the PR #6